client/src/components/WeatherCard.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import './WeatherCard.css'
+
+function convertUnixTimestampToCityTime(unixTimestamp, utcOffsetInSeconds) {
+    const utcSeconds = unixTimestamp + utcOffsetInSeconds;
+    const cityDate = new Date(utcSeconds * 1000);
+
+    return cityDate.toLocaleTimeString(undefined, {
+        timeZone: 'UTC',
+        hour: 'numeric',
+        minute: 'numeric'
+    });
+}
+
+
+
+function WeatherCard({ data, city }) {
+     
+    if (data && city) {
+        return (
+            <div className='weatherCard'>
+                <h3>
+                    <span>{data.name}, {data.sys.country}</span>
+                </h3>
+                <h3>
+                    <span>{Math.round((data.main.temp-273.15) * 9/5 + 32)} Fahrenheit, {Math.round((data.main.temp-273.15))} Celsius </span>
```

Comment:
When dealing with real world units such as time and temperature, you almost never want to use names like "time" or "temp". That lacks the detail needed to know what the unit actually is.

This line is a great example. It clearly tells me that `data.main.temp` is neither Fahrenheit nor Celsius... so is it Kelvin? That should be made obvious at every step along the way, by making every `temp` name be e.g. `temp_k` or similar.

Not doing that is how things like https://www.simscale.com/blog/nasa-mars-climate-orbiter-metric/ happen. E.g.  

> The software calculated the force that the thrusters needed to exert in pounds of force. A second piece of code that read this data assumed it was in the metric unit—“newtons per square meter”.

Reply:
Noted. OpenWeatherMap API uses "temp" in its JSON so I will leave this for now.